Editor/EditorCore/SceneDragAndDropListener.cs

```diff
@@ -47,7 +47,7 @@ public static bool isDragging
 
         static bool isFaceMode
         {
-            get { return ProBuilderEditor.selectMode == SelectMode.Face; }
+            get { return ProBuilderEditor.selectMode == SelectMode.Face || ProBuilderEditor.selectMode == SelectMode.TextureFace; }
```

Comment:
@MartinImgSpc noticed that in TextureTool.cs, isFaceMode checks if selectMode contains flags instead of checking if it matches a single flag like here:
```
var isFaceMode = ProBuilderEditor.selectMode.ContainsFlag(SelectMode.TextureFace | SelectMode.Face);
```
Would this be preferable? Let me know, thanks!

Reply:
Either one is fine, though personally I prefer the `ContainsFlag` version.